### DISCUSSION TITLE
Components documentation

### DISCUSSION THREAD

**Discussion by adamalfredsson** [payloadcms/payload#3055] [Q&A]
Hi! I'm following the video rich text element implementation from the demo repository, which is importing components from payloadcms:

```typescript
import { Button, MinimalTemplate, X } from "payload/components";
import { Form, Select, Submit, Text } from "payload/components/forms";
import { ElementButton } from "payload/components/rich-text";
```

I'm trying to find documentation about these imports, but I'm unable to find anything on the documentation website. Does any exist?

**Comment by JessRynkar** [comment]
Hi @adamalfredsson - this does not exist currently but it is on our roadmap! We are working on a reusable Payload component library supported with documentation on how to use them. Keep an eye on our progress here 👉 https://github.com/payloadcms/payload/discussions/293 

In the meantime, what information were you looking for regarding these components? I'll try and get you some answers.
